src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift

```diff
@@ -105,7 +105,7 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
 							style: .headline,
 							color: .enaColor(for: .textTint),
 							title: AppStrings.ExposureSubmissionHotline.phoneNumberDomestic,
-							accessibilityLabel: AppStrings.ExposureSubmissionHotline.callButtonTitle,
+							accessibilityLabel: "\(AppStrings.ExposureSubmissionHotline.callButtonTitle):\n\n\(AppStrings.ExposureSubmissionHotline.phoneNumberDomestic)",
```

Comment:
I wonder about formatting here. Won't it make more sense to adde a new AppString?
And as @ArturFriesen  mentioned I also wonder about the newlines.

Reply:
@KaiTeuber I removed the line breaks, I would suggest to use the current strings and not create a new string, reason: if we need to change the number, we have to do at two different places and if we forget to change then the voice over will be different to that of the exact number.